apps/site/app/[locale]/next-data/og/route.tsx

```diff
@@ -60,20 +55,16 @@ export const generateStaticParams = async () => [
   { locale: defaultLocale.code },
 ];
 
-// We want to use `edge` runtime when using Vercel
-// @see https://nextjs.org/docs/app/api-reference/file-conventions/route-segment-config#runtime
-export const runtime = VERCEL_ENV ? 'edge' : 'nodejs';
-
 // In this case we want to catch-all possible requests. This ensures that we always generate and
 // serve the OpenGrapgh images independently on the locale
 // @see https://nextjs.org/docs/app/api-reference/file-conventions/route-segment-config#dynamicparams
 export const dynamicParams = true;
 
 // Enforces that this route is used as static rendering
 // @see https://nextjs.org/docs/app/api-reference/file-conventions/route-segment-config#dynamic
-export const dynamic = ENABLE_STATIC_EXPORT ? 'force-static' : 'auto';
+export const dynamic = 'auto';
```

Comment:
@bmuenzenmeyer we have an issue with this because this is the only endpoint failing build for static exports.

This **endpoint** is just incompatible with static exports. What now?

Reply:
This should also mean this whole time OG generation was supposed to be broken for static exports... Which we shouldn't break.

What should we do here?

Reply:
i see this had since changed - i am just now looking at it - lemme know if it's still a concern. the original implementation of next og work referenced tutorials that didn't really help much as it pertained to our combinations of use cases

Reply:
It is still a concern.

Reply:
cc @bmuenzenmeyer

Reply:
thanks for fixing this!